src/main/java/com/uber/jenkins/phabricator/CommentBuilder.java

```diff
@@ -89,9 +95,16 @@ public void processParentCoverage(CodeCoverageMetrics parentCoverage, String bas
             comment.append("Coverage remained the same (" + lineCoverageDisplay + "%)");
         }
 
+        // If coverage goes below a certain threshold fail the build
+        if (coverageDelta < coverageFailThreshold) {
```

Comment:
So coverageDelta can be positive too. Since `coverageFailThreshold` is a max coverage dip percentage, should we only make this check if the delta is negative?